dependencies.yaml

```diff
@@ -470,6 +470,7 @@ dependencies:
     common:
       - output_types: [conda, requirements]
         packages:
+          - certifi
```

Comment:
@nv-rliu Can I ask why this was added to notebook dependencies? It looks like the package is only imported in tests, not notebooks.

Reply:
Ah, my apologies. I thought that we needed this package in the notebook tests because we also use the datasets API in the notebooks a lot. It may not be necessary after all..

Reply:
I don't know if "having certifi installed" is sufficient to avoid this issue or not. Were you seeing certificate failures on notebook jobs before? Are they resolved now?

Reply:
You're right. Just having it installed doesn't actually do anything for the notebooks. 

I am not seeing any SSL failures for the notebooks so we can remove this dependency. Should I go ahead and do that in 24.12 or should it be in 25.02?

Reply:
25.02 please!